File: CS50X/Week6/mario.py
while True:
    height = input("Height: ")
    try:
        height = int(height)
    except:
        continue
    if height > 8:
        continue
    break

brick = "#"
space = " "
n = height

while True:
    print(space * (n - 1), end="")
    print(brick * (height - n + 1), end="")
    n -= 1
    print("  ", end="")
    print(brick * (height - n))
    if n == 0:
        break

File: CS50X/Week0-1_C/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int amount;
    do
    {
        amount = get_int("Height: ");
    }
    while (amount < 1 || amount > 8);

    for (int height = 1; height < amount + 1; height++)
    {
        int distance = amount - height;
        for (int space = distance; space > 0; space--)
        {
            printf(" ");
        }

        for (int c = 0; c < height; c++)
        {
            printf("#");
        }
        printf("  ");
        for (int width = 0; width < height; width++)
        {
            printf("#");
        }

        printf("\n");
    }
}
